__tests__/album-delete.test.js

```diff
@@ -0,0 +1,90 @@
+const { expect } = require('chai');
+const request = require('supertest');
+
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('delete album', () => {
+  let db;
+  let albums;
+  beforeEach(async () => {
+    db = await getDb();
+
+    await Promise.all([
+      db.query('INSERT INTO Artist (name, genre) VALUES(?, ?)', [
+        'Radiohead',
+        'rock'
+      ]),
+      db.query('INSERT INTO Artist (name, genre) VALUES(?, ?)', [
+        'Gorillaz',
+        'pop'
+      ]),
+      db.query('INSERT INTO Artist (name, genre) VALUES(?, ?)', [
+        'Red Hot Chili Peppers',
+        'rock'
+      ])
+    ]);
+
+    const [[radiohead]] = await db.query(
+      'SELECT id FROM Artist WHERE name = ?',
+      ['Radiohead']
+    );
+    const [[gorillaz]] = await db.query(
+      'SELECT id FROM Artist WHERE name = ?',
+      ['Gorillaz']
+    );
+    const [[rhcp]] = await db.query('SELECT id FROM Artist WHERE name = ?', [
+      'Red Hot Chili Peppers'
+    ]);
+
+    await Promise.all([
+      db.query('INSERT INTO Album (name, year, artistId) VALUES (?, ?, ?)', [
+        'OK Computer',
+        1997,
+        radiohead.id
+      ]),
+      db.query('INSERT INTO Album (name, year, artistId) VALUES (?, ?, ?)', [
+        'Plastic Beach',
+        2010,
+        gorillaz.id
+      ]),
+      db.query('INSERT INTO Album (name, year, artistId) VALUES (?, ?, ?)', [
+        'Californication',
+        1999,
+        rhcp.id
+      ])
+    ]);
+
+    [albums] = await db.query('SELECT * from Album');
+  });
+
+  afterEach(async () => {
+    await db.query('DELETE FROM Album');
+    await db.query('DELETE FROM Artist');
+    await db.close();
+  });
+
+  describe('/album/:albumId', () => {
+    describe('DELETE', () => {
+      it('deletes a single album with the correct id', async () => {
+        const album = albums[0];
+        const res = await request(app).delete(`/album/${album.id}`).send();
+
+        expect(res.status).to.equal(200);
```

Comment:
This line could removed, as it isnt being returned in the test output and is just checking part of the setup for the test worked (which would be confirmed if the assertion on line 80 returned as expected anyway). Was this part of debugging when writing the test?

Reply:
Yeah it was just a part of the initial test to make sure the endpoint was set up correctly :+1: